posthog/models/session_recording_event/util.py

```diff
@@ -52,6 +52,18 @@ def create_session_recording_event(
     return str(uuid)
 
 
+def get_recording_count_for_team(team_id: Union[str, int]) -> int:
```

Comment:
I had left this one out as it wouldn't really be a lifetime value, but only what's in storage before expiration

Reply:
Yeah I think its fine tbh. I'll change the key to be "total" so it is not confused with actual "lifetime". But still useful to have as then we can get an idea of total usage on self-hosted for example